# Emit /404.html so GitHub pages loads this instead of its own 404 page
# Initially applied to Digital Garden template version 1.91.0.

diff --git a/src/site/404.njk b/src/site/404.njk
--- a/src/site/404.njk
+++ b/src/site/404.njk
@@ -1,4 +1,6 @@
-
+---
+permalink: 404.html
+---
 <!DOCTYPE html>
 <html lang="{{ meta.mainLanguage }}">
     <head>
@@ -19,8 +21,9 @@
             {%-if not meta.themeStyle%}
                 <div class="font-bg"> &#x1F60E; </div>
             {%endif%}
+            <h1>404</h1>
             <h1>There is nothing here</h1>
-            <p>If you got here from a link, this note is probably not made public</p>
+            <p>If you got here from a link, this note is probably not made (or no longer) public</p>
             <a href="/">Go back home</a>
         </div>
     </body>
